server/athenian/api/controllers/contributors_controller.py

```diff
@@ -0,0 +1,10 @@
+from aiohttp import web
+
+
+async def get_contributors(request: web.Request, id: int) -> web.Response:
+    """List all the contributors belonging to the specified account.
```

Comment:
We should really-really post this in a separate PR. This is becoming an unmanageable monster.

Reply:
Do you mind if we keep just the spec and the stub? The idea is that with this version of the spec everything needed for the settings page is ready. Then after I finished the implementation I'll move to change the filtering.

Reply:
This is actually also the first endpoint that I implemented that just needs to be opened 😄